capacity_by_user.py

```diff
@@ -143,15 +143,14 @@ def get_file_attrs(x):
     client.login(credentials["user"], credentials["password"])
     result = []
     for path in paths:
-        if seen.has_key(path):
+        if file_id in seen:
             result += [seen[path]]
             continue
         attrs = client.fs.get_file_attr(path)
         str_owner = translate_owner_to_owner_string(client
-                                                          , attrs['owner']
-                                                          , attrs['owner_details']['id_type']
-                                                          , attrs['owner_details']['id_value'])
-        seen[path] = str_owner
+                                                   , attrs['owner']
+                                                   , attrs['owner_details']['id_type']
+                                                   , attrs['owner_details']['id_value'])
```

Comment:
this line got deleted! I think unintentionally
```suggestion
                                                   , attrs['owner_details']['id_value'])
            seen[path] = str_owner
```

Reply:
Ay, good catch! Incidentally, the "seen" object seems like needless 'optimization'. The code is trying to sample a usually-small number of files in a large filesystem, but the "seen" object is there in case you happen to see the same file more than once!

OTOH, The program runs slowly because it has to make a separate get_file_attrs request for each file_id, and that ends up being a lot of latency. So :feature_request:, make a version of the get_file_attrs API call that takes a batch of file_ids to lookup all at once.

Reply:
FYI @cwboden these thoughts for https://github.com/Qumulo/capacity-by-user/pull/7